pkg/api/types.go

```diff
@@ -977,10 +977,13 @@ type MemoryBackedVolume struct {
 type ContainerTestConfiguration struct {
 	// From is the image stream tag in the pipeline to run this
 	// command in.
-	From PipelineImageStreamTagReference `json:"from"`
+	From PipelineImageStreamTagReference `json:"from,omitempty"`
```

Comment:
This isn't optional, though?

Reply:
remainder from a previous approach, fixed